### DISCUSSION TITLE
Log to file

### DISCUSSION THREAD

**Discussion by hephaestus9** [pthom/imgui_bundle#192] [Q&A] [answered]
Is it possible to use the logger to log to a file in addition to the log window?

**Comment by pthom** [comment, marked as answer]
Hello,

The API for logging is actually quite simple, and if you want to log to a file and/or stdout in addition to the log window, my advice is to use a logging library on top of HelloImGui's logging facility. 

A logging library such as [spldlog](https://github.com/gabime/spdlog) in C++ or such as the [logging module](https://docs.python.org/3/library/logging.html) in python offers the possibility to log to different "sinks" (i.e. outputs).

I asked ChatGPT for advices on this subject using these libraries, and its output might be a good starting point:

https://chat.openai.com/share/fad96ee4-d9be-4cfc-b98e-803ecdd90a7b

**Reply by hephaestus9** [reply]
Thanks for the quick response!
